### PR TITLE
add getting started section per #189

### PR DESCRIPTION
**Pull request by cglewis**

### CODE REVIEW

**Comment by eddywashere** [comment]
Is this a work in progress pr? If so, please prefix the title with `[WIP]`. Otherwise, I noticed a few references leftover from react-bootstrap and bootstrap 3.

**Comment by cglewis** [comment]
sorry about that, see if that looks better now.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,72 @@ Import the components you need, example:
...
+import React, { Component } from 'react';
+import { Grid, Navbar, Jumbotron, Button } from 'reactstrap';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <Navbar inverse fixedTop>
+          <Grid>
+            <Navbar.Header>
```
Comment (eddywashere) [inline README.md]: The gist content will need to be updated to use the component API according to the reactstrap docs. See https://reactstrap.github.io/components/navbar/

Same goes for the button example.

File: README.md [hunk 2]
```diff
@@ -21,6 +21,72 @@ Import the components you need, example:
...
 
+## Getting Started
+
+You don’t have to use [Reactstrap](https://reactstrap.github.io) together with React but it is a popular library for integrating Bootstrap with React apps. If you need it, you can integrate it with Create React App by following these steps:
+
+Install Reactstrap and Bootstrap from NPM. Reactstrap does not include Bootstrap CSS so this needs to be installed as well:
+
+```
+npm install react-bootstrap --save
+npm install bootstrap@4 --save
```
Comment (eddywashere) [inline README.md]: Should be npm install bootstrap@4.0.0-alpha.5

File: README.md [hunk 3]
```diff
@@ -21,6 +21,72 @@ Import the components you need, example:
...
+```
+
+Import Bootstrap CSS and optionally Bootstrap theme CSS in the ```src/index.js``` file:
+
+```js
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/dist/css/bootstrap-theme.css';
```
Comment (eddywashere) [inline README.md]: This theme CSS can be removed

File: README.md (hunk 1 above)
Comment (eddywashere) [inline README.md]: Component template has an example of what the navbar looks like https://github.com/reactstrap/component-template/blob/master/src/docs/layout/Nav.js

File: README.md [hunk 4]
```diff
@@ -21,6 +21,72 @@ Import the components you need, example:
...
+          <Grid>
+            <Navbar.Header>
+              <Navbar.Brand>
+                <a href="/">React App</a>
+              </Navbar.Brand>
+              <Navbar.Toggle />
+            </Navbar.Header>
+          </Grid>
+        </Navbar>
+        <Jumbotron>
```
Comment (eddywashere) [inline README.md]: Here's what it could look like for jumbotron section https://github.com/reactstrap/component-template/blob/master/src/docs/Home.js#L30

**Comment by cglewis** [comment]
not sure if I did the button correctly with the link?

**Comment by cglewis** [comment]
I think the build failed due to the dns outage yesterday.

File: README.md [hunk 5]
```diff
@@ -21,6 +21,70 @@ Import the components you need, example:
...
+```js
+import React, { Component } from 'react';
+import { Grid, Navbar, Jumbotron, Button } from 'reactstrap';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <Navbar color="faded" light>
+          <NavbarBrand href="/">reactstrap</NavbarBrand>
```
Comment (TheSharpieOne) [inline README.md]: `NavbarBrand` should be added to the import

File: README.md [hunk 6]
```diff
@@ -21,6 +21,70 @@ Import the components you need, example:
...
+            <NavItem>
+              <NavLink href="/components/">Components</NavLink>
+            </NavItem>
+            <NavItem>
+              <NavLink href="https://github.com/reactstrap/reactstrap">Github</NavLink>
+            </NavItem>
+          </Nav>
+        </Navbar>
+        <Jumbotron>
+          <Grid>
```
Comment (TheSharpieOne) [inline README.md]: I don't believe  `Grid` is part of reactstrap. You probably want `Container` or `Row` and `Col`.

File: README.md [hunk 7]
```diff
@@ -21,6 +21,70 @@ Import the components you need, example:
...
+            <h1>Welcome to Reactstrap</h1>
+            <p>
+              <Button
+                color="success"
+                size="lg">
+                View Reactstrap Docs
+              </Button>{'http://reactstrap.github.io/components/buttons/'}
```
Comment (TheSharpieOne) [inline README.md]: Add the URL to the `Button` as a `href` and add the `tag` prop with the value `a` (`tag="a"`) to make it an anchor/link.

**Comment by eddywashere** [comment]
closing this in favor of #291

**Pull request closed without merging**